pkg/logging/logger.go

```diff
@@ -0,0 +1,80 @@
+package logging
```

Comment:
I just noticed that none of the files have the preamble such as https://github.com/pulumi/pulumi/blob/master/sdk/go/auto/stack_test.go#L1

Reply:
Yeah we recently added a check for this in pulumictl https://github.com/pulumi/pulumictl/tree/master/cmd/pulumictl/copyright. This repo hasn't been hooked up to it yet. Will do that separately.